src/config.ml

```diff
@@ -1,22 +1,14 @@
 open Mirage
 
-let suspend =
```

Comment:
This is used later on when we look for it in the cmdline arguments.

Is there an equivalent with the new mirage version?

Reply:
Although I can't actually find where... maybe it was dead code?

Reply:
There's an equivalent to have runtime arguments, but i couldn't see where it was being used